### PR TITLE
Better log processing

### PR DESCRIPTION
**Pull request by boboldehampsink**
Untested tho

### CODE REVIEW

**Comment by timkelty** [comment]
Yep looks good to me.
Could also stick `levels` in as a plugin config for overriding.

**Comment by boboldehampsink** [comment]
Will do that in the future - will make it so that you can also override them for file logs then (by removing the file route and adding an extended file route with new init options).

**Comment by boboldehampsink** [comment]
Can you help me test this?

**Comment by timkelty** [comment]
yep, will do now.

**Comment by timkelty** [comment]
Tested this branch - seems to block all logging. :(
Didn't notice any errors. Can dig in more tomorrow.

**Comment by boboldehampsink** [comment]
Are you running in devMode?

**Comment by timkelty** [comment]
Tried both ways. Seems that `$logs = parent::processLogs($logs);` is always returning an empty array.

File: logging/LogHelper_BaseLogRoute.php [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    /**
+     * Saves log messages in files.
+     *
+     * @param array $logs The list of log messages
+     *
+     * @return null
+     */
+    protected function processLogs($logs)
+    {
+        $logs = array();
```
Comment (timkelty) [inline logging/LogHelper_BaseLogRoute.php]: @boboldehampsink this would be the culprit, I believe :)

**Comment by boboldehampsink** [comment]
Oops, fixed that

**Comment by timkelty** [comment]
OK, test that, seems to be working (including devMode toggling info/warnings)

**Comment by timkelty** [comment]
eh?

**Pull request merged**